apps/system/lockscreen/js/lockscreen_charging.js

```diff
@@ -39,47 +43,48 @@ LockScreenChargingStatus.prototype.handleEvent = function cs_handleEvent(evt) {
 };
 
 LockScreenChargingStatus.prototype.refresh = function cs_refresh() {
-
-  if (!navigator.battery.charging) {
-    if (!this.elements.charging.hasAttribute('hidden')) {
-      this.elements.charging.setAttribute('hidden', '');
-    }
-    return;
-  }
-
-  var l10nAttrs = {
-    id: null,
-    args: {
-      level: parseInt(navigator.battery.level * 100)
+  navigator.getBattery().then(battery => {
+    if (!battery.charging) {
+      if (!this.elements.charging.hasAttribute('hidden')) {
+        this.elements.charging.setAttribute('hidden', '');
+      }
+      return;
     }
-  };
 
-  var chargingTime = navigator.battery.chargingTime;
+    var l10nAttrs = {
+      id: null,
+      args: {
+        level: parseInt(battery.level * 100)
+      }
+    };
 
-  if (chargingTime === Infinity ||
-      chargingTime < 60) { // less than a minute remaining
-    l10nAttrs.id = 'charging-no-time';
-  } else {
-    var timeLeft = new Date(0, 0, 0, 0, 0, chargingTime);
-    l10nAttrs.args.hours = timeLeft.getHours();
-    l10nAttrs.args.minutes = timeLeft.getMinutes();
+    var chargingTime = battery.chargingTime;
 
-    if (l10nAttrs.args.hours > 0) {
-      l10nAttrs.id = 'charging-hours';
+    if (chargingTime === Infinity ||
+        chargingTime < 60) { // less than a minute remaining
+      l10nAttrs.id = 'charging-no-time';
     } else {
-      l10nAttrs.id = 'charging-minutes';
+      var timeLeft = new Date(0, 0, 0, 0, 0, chargingTime);
+      l10nAttrs.args.hours = timeLeft.getHours();
+      l10nAttrs.args.minutes = timeLeft.getMinutes();
+
+      if (l10nAttrs.args.hours > 0) {
+        l10nAttrs.id = 'charging-hours';
+      } else {
+        l10nAttrs.id = 'charging-minutes';
+      }
     }
-  }
 
-  // Update data here since changing textContent will replace the text node and
-  // could cause flickering while transitioning docshell.isActive to true.
-  document.l10n.formatValue(l10nAttrs.id, l10nAttrs.args).then((value) => {
-    this.elements.charging.firstChild.data = value;
-  });
+    // Update data here since changing textContent will replace the text node 
+    // and could cause flickering while transitioning docshell.isActive to true.
+    document.l10n.formatValue(l10nAttrs.id, l10nAttrs.args).then((value) => {
+      this.elements.charging.firstChild.data = value;
+    });
```

Comment:
same here. If you look at the code, the indentation is correct. It just looks weird in this diff.